sdk/metricsadvisor/ai-metrics-advisor/review/ai-metrics-advisor.api.md

```diff
@@ -438,18 +433,7 @@ export type GetMetricEnrichedSeriesDataResponse = {
 };
 
 // @public
-export type GetMetricFeedbackResponse = {
-    body: MetricFeedbackUnion;
-    _response: coreHttp.HttpResponse & {
-        bodyAsText: string;
-        parsedBody: MetricFeedbackUnion;
-    };
-};
-
-// @public
-export type GetMetricSeriesDataOptions = {
-    skip?: number;
```

Comment:
@jeremymeng  What is `skip` used for?

Reply:
It's not used anywhere, probably a leftover from copy and paste.